include/ShearStressTransportEquationSystem.h

```diff
@@ -92,7 +92,7 @@ class ShearStressTransportEquationSystem : public EquationSystem
 
   const double tkeMinValue_{1.0e-8};
   const double sdrMinValue_{1.0e-8};
-  const double gammaMinValue_{0.0};
+  const double gammaMinValue_{0.02}; // 1/ce2=1/50=0.02
```

Comment:
remove comment and prefer signaling in documentation

Reply:
removed